Assets/Scripts/Entity/Enemy/EnemyDrop.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyDrop : MonoBehaviour
+{
+    /* Called when an enemy has died. Spawns a heart with random probability.
+    */
+    public static void DropHeart(Transform t, float healthDropProbability, float heartTypeProbability)
```

Comment:
It may be better to create `public` or serialized `private` fields in `EnemyDrop.cs` that allow us to set the `healthDropProbability` and `heartTypeProbability` values in the inspector. Otherwise, we'll probably need to add that information to Enemy.cs as fields (not ideal) or hard-code the values when calling `DropHeart` (also not ideal).